tutorials/rendering/viewports.rst

```diff
@@ -95,11 +100,17 @@ these values are overridden, but for all others, this sets their resolution.
 It is also possible to scale the 2D content and make the :ref:`SubViewport <class_SubViewport>` resolution
 different from the one specified in size, by calling:
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     sub_viewport.set_size_2d_override(Vector2i(width, height)) # Custom size for 2D.
     sub_viewport.set_size_2d_override_stretch(true) # Enable stretch for custom size.
 
+ .. code-tab:: csharp
+
+    subViewport.SetSize2DOverride(new Vector2I(width, height));
+    subViewport.SetSize2DOverrideStretch(true);
```

Comment:
Prefer using the properties over the hidden methods. It's also missing the comments from the GDScript code.

```suggestion
    subViewport.Size2DOverride = new Vector2I(width, height); // Custom size for 2D.
    subViewport.Size2DOverrideStretch = true; // Enable stretch for custom size.
```